Formula/fgsl.rb

```diff
@@ -0,0 +1,30 @@
+class Fgsl < Formula
+  desc "Fortran bindings for the GNU Scientific Library"
+  homepage "https://www.lrz.de/services/software/mathematik/gsl/fortran/"
+  url "https://www.lrz.de/services/software/mathematik/gsl/fortran/download/fgsl-1.2.0.tar.gz"
+  sha256 "00fd467af2bb778e8d15ac8c27ddc7b9024bb8fa2f950a868d9d24b6086e5ca7"
+
+  depends_on "pkg-config" => :build
+  depends_on "gcc"
+  depends_on :fortran
+  depends_on "gsl"
+
+  def install
+    ENV.deparallelize
```

Comment:
This needs to be reported and fixed upstream before we can merge the formula.

Reply:
Ok... I understand... but most of the fortran sofware are not able to be compiled in parallel.